contracts/DCAPermissionsManager/DCAPermissionsManager.sol

```diff
@@ -173,6 +181,33 @@ contract DCAPermissionsManager is ERC721, EIP712, Governable, IDCAPermissionMana
     _modify(_tokenId, _permissions);
   }
 
+  /// @inheritdoc IDCAPermissionManager
+  function multiPermissionPermit(
+    PositionPermissions[] calldata _permissions,
+    uint256 _deadline,
+    uint8 _v,
+    bytes32 _r,
+    bytes32 _s
+  ) external {
+    if (block.timestamp > _deadline) revert ExpiredDeadline();
+
+    address _owner = ownerOf(_permissions[0].tokenId);
+    bytes32 _structHash = keccak256(abi.encode(MULTI_PERMISSION_PERMIT_TYPEHASH, keccak256(_encode(_permissions)), nonces[_owner]++, _deadline));
+    bytes32 _hash = _hashTypedDataV4(_structHash);
+
+    address _signer = ECDSA.recover(_hash, _v, _r, _s);
+    if (_signer != _owner) revert InvalidSignature();
+
+    for (uint256 i = 0; i < _permissions.length; i++) {
+      uint256 _tokenId = _permissions[i].tokenId;
+      if (i > 0) {
+        address _positionOwner = ownerOf(_tokenId);
+        if (_owner != _positionOwner) revert NotOwner();
```

Comment:
```suggestion
        if (_signer != _positionOwner) revert NotOwner();
```
Maybe this is kinda weird, and I do understand that is the same thing, since previously we've stated that `_signer == _owner` but it's kinda confusing, since `_owner` is derived from the first position. Which can be tricky. 
Just a suggestion.